### ISSUE TITLE
Audio should allow repeat-track mode

### ISSUE THREAD

**Issue by oakbrad** [Cog-Creators/Red-DiscordBot#6467]
### What component of Red (cog, command, API) would you like to see improvements on?

audio

### Describe the enhancement you're suggesting.

Most audio players have two repeat modes: repeat, and repeat-track.

Currently, Red's audio bot has a repeat command that repeats the current playlist.

It would be wonderful if Red also had a repeat-track mode, where the current track is repeated until the user moves the playlist position manually.

**Use Case:** I run online RPG games and we use Red for background ambiance for each scene. Ideally I would be able to make a playlist for the game, set to repeat-track, and then manually move the playlist position for each scene. Currently my work around is to make playlists with 1 track each, but that is more difficult to manage while playing.

**Comment by japandotorg** [comment]
Nope, Red currently doesn't have the ability to loop/repeat a single song. However you can make a queue with a single song and have that on repeat.

Although i'd love to see a "repeat track" feature, +1 from me.

**Comment by CGavinMullis** [comment]
Also +1 for me, for tabletop roleplaying, looping a single track is often needed, but I still want to be able to queue up the next song, without needing to do multiple commands back to back

- play first song
- enable repeat

then when I want to play the next song

- disable repeat
- play next song
- skip
- enable repeat

**Comment by ViroAI** [comment]
Hey there! Adding a 'repeat-track' mode to the Audio cog is a great enhancement. I specialize in Discord bot audio implementations and Cog development. I can build this feature for you, ensuring it integrates perfectly with the current Red-Bot audio queue system. I work via Fiverr to ensure professional quality and support. Check out my work here: https://www.fiverr.com/s/1qZlNze. Let me know if you want me to start on this!
